nextflow.config

```diff
@@ -538,8 +538,8 @@ params {
     }
 
     staramr {
-        singularity = "https://depot.galaxyproject.org/singularity/staramr%3A0.9.1--pyhdfd78af_0"
-        docker = "quay.io/biocontainers/staramr:0.9.1--pyhdfd78af_0"
+        singularity = "quay.io/biocontainers/staramr:0.10.0--pyhdfd78af_0"
```

Comment:
There is no singularity container for staramr 0.10.0 in depot.galaxyproject.org so I'm using the Docker container, which should work with singularity.